src/templates/vasculature_class.owl

```diff
@@ -81,7 +81,7 @@
                 <owl:someValuesFrom rdf:resource="http://purl.obolibrary.org/obo/UBERON_0001142"/>
             </owl:Restriction>
         </owl:annotatedTarget>
-        <oboInOwl:inSubset rdf:resource="http://purl.obolibrary.org/obo/uberon/core#human_reference_atlas"/>
+        <oboInOwl:inSubset rdf:resource="&lt;http://purl.obolibrary.org/obo/uberon/core#human_reference_atlas&gt;"/>
```

Comment:
@dosumis it was already good before changing to the complete URI; somewhere it's already defined (?)

Reply:
I'll revert the commit.